Dockerfile

```diff
@@ -1,4 +1,4 @@
-# syntax=docker/dockerfile:1
+blem# syntax=docker/dockerfile:1
```

Comment:
this appears to be unintended extra text

Reply:
this has been removed - thank you for catching it!